feedhenry/fh-cachekey-client
# task(CEV): RHMAP-20775 - nodejs-bl: Numeric arguments passed to bl.append() return uninitialized memory

# Jira link(s)
https://issues.jboss.org/browse/RHMAP-20775

# What
* Solve CEV related to the [bl](https://github.com/rvagg/bl) lib.

# Why
* To fix CEV

# How
* Upgrade version of request lib from `^2.53.0`to `2.87.0`
* Run `npm install --save --save-exact request@2.87.0`

# Verification Steps
* Check if the CI process will run the unit tests will finish with success. 
* No break changes on requestJS lib
